cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -189,6 +189,7 @@ private void initialize(@NonNull Context context, @Nullable AttributeSet attrs)
                 0);
         int videoBitRate = a.getInteger(R.styleable.CameraView_cameraVideoBitRate, 0);
         int audioBitRate = a.getInteger(R.styleable.CameraView_cameraAudioBitRate, 0);
+        float videoFrameRate = a.getFloat(R.styleable.CameraView_cameraVideoFrameRate, 30);
```

Comment:
Can you default to 0F instead ? It will have the meaning of "let engine choose". See comments later